compass/ocean/tests/utility/combine_topo/__init__.py

```diff
@@ -158,169 +180,410 @@ def _modify_bedmachine(self):
+        if lat_tile == 0:
+            lat_indices = np.insert(lat_indices, 0, 0)
+        if lat_tile == lat_tiles - 1:
+            lat_indices = np.append(lat_indices, lat_indices[-1])
+
+        # Select tile from GEBCO
+        tile = gebco.isel(lat=lat_indices, lon=lon_indices)
+        if lat_tile == 0:
+            tile.lat.values[0] = -90.  # Correct south pole
+        if lat_tile == lat_tiles - 1:
+            tile.lat.values[-1] = 90.  # Correct north pole
+
+        # Write tile to netCDF
+        tile.to_netcdf(out_filename)
+
+    def _create_bedmachine_scrip_file(self):
+        """
+        Create SCRIP file for BedMachineAntarctica data using pyremap
+        """
+        logger = self.logger
+        logger.info('    Creating BedMachine SCRIP file')
+
+        # Parse config
+        config = self.config
+        section = config['combine_topo']
+        antarctic_name = section.get('antarctic_filename').strip('.nc')
+        in_filename = f'{antarctic_name}_mod.nc'
+        out_filename = f'{antarctic_name}.scrip.nc'
+
+        # Define projection
+        projection = pyproj.Proj(
+            '+proj=stere +lat_ts=-71.0 +lat_0=-90 +lon_0=0.0 '
+            '+k_0=1.0 +x_0=0.0 +y_0=0.0 +ellps=WGS84'
+        )
+
+        # Create SCRIP file using pyremap
+        bedmachine_descriptor = ProjectionGridDescriptor.read(
+            projection, in_filename, 'BedMachineAntarctica500m',
+        )
+        bedmachine_descriptor.to_scrip(out_filename)
+
+    def _create_target_scrip_file(self):
         """
-        Remap BedMachine Antarctica to GEBCO lat-lon grid
+        Create SCRIP file for either the x.xxxx degree (lat-lon) mesh or the
+        NExxx (cubed-sphere) mesh, depending on the value of `self.target_grid`
+        References:
+          https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/872579110/
+          Running+E3SM+on+New+Atmosphere+Grids
         """
         logger = self.logger
-        logger.info('Remap BedMachineAntarctica to GEBCO 1/80 deg grid')
+        logger.info(f'Create SCRIP file for {self.resolution_name} mesh')
+
+        out_filename = self.outputs[0]
+
+        # Build cubed sphere SCRIP file using tempestremap
+        if self.target_grid == 'cubed_sphere':
+
+            # Create EXODUS file
+            args = [
+                'GenerateCSMesh', '--alt', '--res', f'{self.resolution}',
+                '--file', f'{self.resolution_name}.g',
+            ]
+            check_call(args, logger)
+
+            # Create SCRIP file
+            args = [
+                'ConvertMeshToSCRIP', '--in', f'{self.resolution_name}.g',
+                '--out', out_filename,
+            ]
+            check_call(args, logger)
+
+        # Build lat-lon SCRIP file using pyremap
+        elif self.target_grid == 'lat_lon':
+            descriptor = get_lat_lon_descriptor(
+                dLon=self.resolution, dLat=self.resolution,
+            )
+            descriptor.to_scrip(out_filename)
```

Comment:
Need to make sure this is in NETCDF3 format.  This should be possible by setting the `.format` argument in the descriptor (see pyremap for details).